src/app.ts

```diff
@@ -13,10 +14,15 @@ import "./app.css";
 const companyId = 1361;
 const floorId = "m2033670";
 
+// Set date for floor plan data
+// Note that date is optional in API requests
+const date = startOfToday();
```

Comment:
Maybe `const date = new Date()` instead. Empty constructor defaults to today. Using vanilla functions is easier to understand.

Reply:
Fixed